### PR TITLE
Jonfritz patch docsupdate

### PR DESCRIPTION
**Pull request by jonfritz**
Added some info to the Aryn Partitioning Service section.

### CODE REVIEW

File: docs/source/aryn_cloud/aryn_partitioning_service.md [hunk 1]
```diff
@@ -102,6 +107,8 @@ The available options are listed below:
 * ```use_ocr```: If ```true```, the partitioner uses OCR to extract text from the PDF. It defaults to ```false```, where the partitioner attempts to directly extract the text from the underlying PDF in the bounding box. It currently uses Tesseract for extraction.
 * `extract_table_structure`: If `true`, the partitioner runs a separate table extraction model to extract cells from regions of the document identified as tables.
 * `extract_images`: If `true`, the partitioner crops each region identified as an image and attaches it to the associated `ImageElement`. This can later be fed into the `SummarizeImages` transform when used within Sycamore.
+* `selected_pages`: You can specify a page (like [11]) or a page range (like [25,30]) of your PDF to process.
```
Comment (HenryL27) [inline docs/source/aryn_cloud/aryn_partitioning_service.md:110]: I'd add here that the first page is 1, not 0

File: docs/source/aryn_cloud/aryn_partitioning_service.md [hunk 2]
```diff
@@ -102,6 +107,8 @@ The available options are listed below:
...
+* `pages_per_call`: This is only available when using the Partition function in Sycamore. This option divides the processing of your document into batches of pages, and you specify the size of each batch. This is useful when running OCR on large documents.
```
Comment (HenryL27) [inline docs/source/aryn_cloud/aryn_partitioning_service.md:111]: If this is only applicable in sycamore, maybe put it in the sycamore section?

**Review by HenryL27 — APPROVED** [review]
Few nits, but won't block

File: docs/source/aryn_cloud/aryn_partitioning_service.md (hunk 1 above)
Comment (jonfritz) [reply docs/source/aryn_cloud/aryn_partitioning_service.md:110]: Added.

File: docs/source/aryn_cloud/aryn_partitioning_service.md (hunk 2 above)
Comment (jonfritz) [reply docs/source/aryn_cloud/aryn_partitioning_service.md:111]: People might not know where to look, and just jump to Options. I think it's most discoverable here, but will also add under the Sycamore section.

**Pull request merged**